alert-database/src/test/java/com/synopsys/integration/alert/database/api/DefaultConfigurationAccessorTest.java

```diff
@@ -0,0 +1,701 @@
+        descriptorConfigEntity.setId(5L);
+
+        setupCreateJobMocks(registeredDescriptorEntity, configContextEntity, definedFieldEntity, descriptorConfigEntity);
+
+        DefaultConfigurationAccessor configurationAccessor = new DefaultConfigurationAccessor(registeredDescriptorRepository, null, definedFieldRepository, descriptorConfigRepository, configGroupRepository, configContextRepository,
+            fieldValueRepository, null);
+        ConfigurationJobModel configurationJobModel = configurationAccessor.createJob(descriptorNames, configuredFields);
+
+        assertEquals(fieldValue, configurationJobModel.getName());
+    }
+
+    @Test
+    public void updateJobTest() throws Exception {
+        UUID uuid = UUID.randomUUID();
+        List<String> descriptorNames = List.of("descriptor-name-test");
+
+        ConfigurationFieldModel configurationFieldModel = ConfigurationFieldModel.create("channel.common.name");
+        configurationFieldModel.setFieldValue(fieldValue);
+        List<ConfigurationFieldModel> configuredFields = List.of(configurationFieldModel);
+        RegisteredDescriptorEntity registeredDescriptorEntity = new RegisteredDescriptorEntity("name", 1L);
+        registeredDescriptorEntity.setId(2L);
+        ConfigContextEntity configContextEntity = new ConfigContextEntity(configContextEnum.name());
+        configContextEntity.setId(3L);
+        DefinedFieldEntity definedFieldEntity = new DefinedFieldEntity(fieldKey, false);
+        definedFieldEntity.setId(4L);
+        DescriptorConfigEntity descriptorConfigEntity = new DescriptorConfigEntity(5L, 6L, DateUtils.createCurrentDateTimestamp(), DateUtils.createCurrentDateTimestamp());
+        descriptorConfigEntity.setId(5L);
+        ConfigGroupEntity configGroupEntity = new ConfigGroupEntity(6L, uuid);
+
+        Mockito.when(configGroupRepository.findByJobId(Mockito.any())).thenReturn(List.of(configGroupEntity));
+        setupCreateJobMocks(registeredDescriptorEntity, configContextEntity, definedFieldEntity, descriptorConfigEntity);
+
+        DefaultConfigurationAccessor configurationAccessor = new DefaultConfigurationAccessor(registeredDescriptorRepository, null, definedFieldRepository, descriptorConfigRepository, configGroupRepository, configContextRepository,
+            fieldValueRepository, null);
+        ConfigurationJobModel configurationJobModel = configurationAccessor.updateJob(uuid, descriptorNames, configuredFields);
+
+        Mockito.verify(descriptorConfigRepository).deleteById(Mockito.any());
+        assertEquals(uuid, configurationJobModel.getJobId());
+        assertEquals(fieldValue, configurationJobModel.getName());
+    }
+
+    @Test
+    public void updateJobNullIdTest() throws Exception {
+        DefaultConfigurationAccessor configurationAccessor = new DefaultConfigurationAccessor(null, null, null, null, null, null, null, null);
+        try {
+            configurationAccessor.updateJob(null, null, null);
+            fail();
```

Comment:
Can you add messages in all of the `fail()` method calls? With a brief explanation?